pwiz_tools/Skyline/Model/Results/ResultsResources.resx

```diff
@@ -333,7 +333,7 @@ It is recommended that you delete this file so that Skyline can create a new fil
     <value>Discarding chromatograms for '{0}' because the explicit retention time {1} is not between {2} and {3}</value>
   </data>
   <data name="IonMobilityFinder_EvaluateBestIonMobilityValue_need_a_value_for_Transition_Settings___ion_mobility_filtering___Window_type" xml:space="preserve">
-    <value>need a value for  Transition Settings  Ion mobility filtering  Window type</value>
+    <value>need a value for Ion mobility filtering Window type in Transition Settings</value>
```

Comment:
We tend to want to direct people from the top down. So, in sentence form, this would be:
In the Transition Settings - Ion Mobility tab, the Ion mobility filtering, Window type field must have a value.

If I were reviewing all these messages from that perspective, there are more issues with them.